### ISSUE TITLE
pub on crates.io

### ISSUE THREAD

**Issue by aizcutei** [yury/cidre#22]
Just found this wonderful repo, do you have any plan to publish this on crates.io in the future?

**Comment by yury** [comment]
Yep, and I reserved name [already](https://crates.io/crates/cidre). Want to finish my app first, so this crate will be really battle tested in big app.

By the way, how did you find cidre?

**Comment by aizcutei** [comment]
Oh I was looking for a ScrennCaptureKit binding and first find [screencaptuekit-rs](https://github.com/doom-fish/screencapturekit-rs/). But it has some problem after upgrade to MacOS 15, then I find [this](https://github.com/doom-fish/screencapturekit-rs/issues/51#issuecomment-2255265719) one mentioned in the issue.

**Comment by thewh1teagle** [comment]
The best way to achieve stability is through actual usage. Other crates that depend on cidre cannot be published because crates cannot rely on unpublished ones. A beta version on crates.io would be really useful!

**Comment by louis030195** [comment]
same

**Comment by oscartbeaumont** [comment]
If you haven't already noticed just wanted to callout that the docs.rs generation seems to have failed.

**Comment by yury** [comment]
> If you haven't already noticed just wanted to callout that the docs.rs generation seems to have failed.

Fixed in v0.9.1. Thanks.
